scripts/new_project.kts

```diff
@@ -206,7 +207,7 @@ object NewProject {
         process.inputStream.reader().forEachLine { println(it) }
         val exitValue = process.waitFor()
         if (exitValue != 0) {
-            showErrorMessage("❌ Something went wrong!", exitValue)
+            showErrorMessage("❌ Something went wrong! when executing command: $command", exitValue)
```

Comment:
@Tuubz I've updated this line to know what command was executing when it failed. Is it okay for you?

Reply:
Yes, that's great 🚀  Thank you for the update 😁